src/lara.h

```diff
@@ -3054,25 +3054,25 @@ struct Lara : Character {
 
     // VR control
         if (Core::settings.detail.stereo == Core::Settings::STEREO_VR && camera->firstPerson && canFreeRotate()) {
+			
             if (!(input & WALK)) {
-                input &= ~(LEFT | RIGHT);
+                //input &= ~(LEFT | RIGHT); // locks side jumping and movement
             }
-
-            vec3 ang = getAngleAbs(Input::hmd.head.dir().xyz());
-
+            vec3 ang = getAngleAbs(Input::hmd.head.dir().xyz()); // set in main
+			//ang = temp; // let's see how broken this gets
             angle.y = ang.y;
-//            rotFactor = vec2(1.0f);
```

Comment:
In VR mode we control rotation directly by HMD, LEFT and RIGHT produce delayed rotate by animation state, that breaks the effect of immersion and the player becomes sick. You can complete remove commented lines.